src/main/java/it/gov/pagopa/pu/debtpositions/service/SpontaneousFormServiceImpl.java

```diff
@@ -28,4 +32,17 @@ public SpontaneousForm createSpontaneousForm(SpontaneousForm spontaneousForm) {
     }
     return spontaneousFormRepository.save(spontaneousForm);
   }
+
+  @Override
+  public void deleteSpontaneousForm(Long spontaneousFormId) {
+    SpontaneousForm spontaneousForm = spontaneousFormRepository.findById(
+        spontaneousFormId).orElseThrow(
+        () -> new NotFoundException("SpontaneousForm having id "+spontaneousFormId+" not found"));
+
+    long dptoCount = debtPositionTypeOrgRepository.countBySpontaneousFormId(spontaneousFormId);
+    if(dptoCount > 0L){
+      throw new ConflictErrorException("The SpontaneousForm having id "+spontaneousFormId+" is referenced by "+dptoCount+" DebtPositionTypeOrgs");
+    }
+    spontaneousFormRepository.delete(spontaneousForm);
```

Comment:
Has the automated openApi registered the `delete` operation exposed by SpringDataRest on SpontaneousForm entity?
If yes, we should exclude... (we have already did something like when we suppressed some `save` operation)